drizzle/migrations/0005_fix_chat_db_fk_normalization.sql

```
@@ -0,0 +1,73 @@
+    FOREIGN KEY (user_id) REFERENCES public.users(id) ON DELETE CASCADE;
+
+ALTER TABLE public.chat_participants
+  ADD CONSTRAINT chat_participants_user_id_fkey
+    FOREIGN KEY (user_id) REFERENCES public.users(id) ON DELETE CASCADE;
+
+ALTER TABLE public.prompt_generation_jobs
+  ADD CONSTRAINT prompt_generation_jobs_user_id_fkey
+    FOREIGN KEY (user_id) REFERENCES public.users(id) ON DELETE CASCADE;
+
+-- Step 3: Add indexes on user_id columns for query performance
+CREATE INDEX IF NOT EXISTS idx_chats_user_id ON public.chats(user_id);
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)